types/pi-camera/pi-camera-tests.ts

```diff
@@ -47,7 +49,5 @@ camera3.snapDataUrl().then((result) => {
 const widthValue = camera.get('width');
 log(widthValue);
 
-camera.set('fps', 25);
-
-const fpsValue = camera.get('width');
-log(fpsValue);
+// $ExpectError
+camera.get('fsp');
```

Comment:
This might be an unintentional typo instead of the intended value of 'fps'. Kudos!

Reply:
@ataberkylmz wait, i'm removing 'fps' as no longer supported value for the config (should be 'framerate'):
https://github.com/stetsmando/pi-camera/blob/2d4ff71ff7c347b84011d61f73f9f8c453ff23a9/options.json
So am I correct, this cannot be a typo any longer (yes, for existing code, if someone has `fsp`, this change will rise errors in compilation)